src/page/main.js

```diff
@@ -6,8 +6,7 @@ import "./cm.css";
 
 if (import.meta.env.DEV) { // vite feat that only import in dev mode
     const modules = import.meta.glob("../shared/dev.js", {eager: true});
-    // eslint-disable-next-line no-global-assign
-    browser = modules["../shared/dev.js"].browser;
+    window.browser = modules["../shared/dev.js"].browser;
```

Comment:
A

Reply:
- [ ] The @ACTCD ![image](https://github.com/quoid/userscripts/assets/163666499/ef1d9de2-4b97-4fe8-b265-85e30cb1d6ce)y